tests/test_objective_funs.py

```diff
@@ -410,37 +402,25 @@ def test(eq):
 
 @pytest.mark.unit
 def test_derivative_modes():
-    """Test equality of derivatives using batched, blocked, looped methods."""
+    """Test equality of derivatives using batched, looped methods."""
```

Comment:
Because all the sub-objectives now take dicts of params instead of the params themselves it got really messy/wasteful trying to get the individual jacobians for each arg. In practice the blocked method was never really used since it had greater memory usage than looped but was slower than batched.